### PR TITLE
Move Mellanox QoS configuration to config DB

### PR DESCRIPTION
**Pull request by andriymoroz-mlnx**
Signed-off-by: Andriy Moroz <c_andriym@mellanox.com>

**- What I did**
Moved QoS config to config DB

FOR REVIEW AND DISCUSSION
PLEASE DO NOT MERGE
See also:
https://github.com/Azure/sonic-swss-common/pull/158
https://github.com/Azure/sonic-swss/pull/378

### CODE REVIEW

**Comment by andriymoroz-mlnx** [comment]
The beginning is here: https://github.com/Azure/sonic-buildimage/pull/1138

File: device/mellanox/x86_64-mlnx_msn2700-r0/ACS-MSN2700/qos.json [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            "green_min_threshold": "184320"
+        },
+        "AZURE_LOSSLESS": {
+            "wred_green_enable":"true",
+            "wred_yellow_enable":"true",
+            "ecn":"ecn_all",
+            "red_max_threshold":"516096",
+            "red_min_threshold":"516096",
+            "yellow_max_threshold":"516096",
+            "yellow_min_threshold":"516096
```
Comment (stcheng) [inline device/mellanox/x86_64-mlnx_msn2700-r0/ACS-MSN2700/qos.json]: need one more comma

Comment (andriymoroz-mlnx) [reply device/mellanox/x86_64-mlnx_msn2700-r0/ACS-MSN2700/qos.json]: and the closing quotes... thanks

**Comment by andriymoroz-mlnx** [comment]
Added qos config for:
```
platform: x86_64-dell_s6000_s1220-r0, hwsku: Force10-S6000 and Force10-S6000-Q32
platform: x86_64-dell_s6100_c2538-r0, hwsku: Force10-S6100
platform: x86_64-arista_7050_qx32, hwsku: Arista-7050-QX32
```
that's all what is publicly available

**Comment by stcheng** [comment]
@andriymoroz-mlnx sorry i accidentally merged this commit. i've reverted it.
Please reopen a new pull request of this one along with the submodule changes including both swss and swss-common repository.
sorry for the inconvenience
thank  you

**Comment by andriymoroz-mlnx** [comment]
ok, I will reopen it after sub-modules pointers update

**Pull request merged**